src/config/schema.ts

```diff
@@ -219,6 +219,27 @@ export const configSchema = {
       type: "object",
       properties: {
         initialStatus: { type: "string", enum: Object.values(AccountStatus) },
+        customFields: {
+          type: "array",
+          items: {
+            type: "object",
+            properties: {
+              name: { type: "string" },
+              type: {
+                type: "string",
+                enum: ["string", "integer", "float", "boolean"],
+              },
+              defaultValue: { type: ["string", "number", "boolean", "array"] },
+              array: { type: "boolean", default: false },
```

Comment:
I thought we shall not support arrays initially?

Reply:
Removed